src/bootstrap/config.rs

```diff
@@ -330,6 +336,18 @@ impl Config {
             }
         }).unwrap_or_else(|| TomlConfig::default());
 
+        let parse_toolstate = || -> Result<_, Box<::std::error::Error>> {
+            let mut f = File::open("toolstate.toml") // local build
+                .or_else(|_| File::open("../toolstate.toml"))?; // travis build
```

Comment:
This should use the path to the source directory to figure this out rather than guessing files